collection-scripts/targeted_logs

```diff
@@ -66,6 +69,19 @@ for nsvm in ${target_vms[@]}; do
     /usr/bin/oc logs --all-containers --namespace ${ns} ${virtV2VPod} &> "${object_collection_path}/current.log" &
     pwait $max_parallelism
   fi
+
+  for pvc in $ovvps $osvps; do
```

Comment:
that's because metadata.name is also the name of the pvc?

Reply:
yes, but i'm looking about dropping this approach.
in the new approach i will use migration id from the plan. then, i don't need this at all.
anyway, this PR is irrelevant as i continued Martin's one (#57).